### PR TITLE
Fix truffle console script and add notes to readme

### PR DESCRIPTION
**Pull request by bowd**
### Description

Fix the console script so that it can pick-up the contract artefacts and add a readme section on usage

### Tested

Ran the commands

### Backwards compatibility

No issues

### Documentation

Updated readme

### CODE REVIEW

File: packages/protocol/scripts/bash/console.sh [hunk 1]
```diff
@@ -26,4 +26,4 @@ if ! nc -z 127.0.0.1 8545 && [ -z $FORNO ] ; then
   exit 1
 fi
 
-yarn run truffle console --network $NETWORK --build_directory $PWD/build/$NETWORK $FORNO
+yarn run truffle console --network $NETWORK --contracts_build_directory $PWD/build/contracts $FORNO
```
Comment (yorhodes) [inline packages/protocol/scripts/bash/console.sh:29]: this is surprising to me as I think we use this `build_directory` elsewhere
we have been using `release-lib.sh` to populate `build/$TAG`

**Review by yorhodes — APPROVED** [review]
super helpful! this README needs a lot of love

**Review by yorhodes — APPROVED** [review]
super helpful! this README needs a lot of love

File: packages/protocol/scripts/bash/console.sh (hunk 1 above)
Comment (bowd) [reply packages/protocol/scripts/bash/console.sh:29]: Yeah I also saw `build_directory` used in different places, but after consulting docs and some source code this is the only way I could get the artefacts to automatically show up as truffle contracts in the global scope in the console. 🤷

**Pull request merged**